articles/understanding-static-dynamic-routing/index.md

```diff
@@ -0,0 +1,276 @@
+# Understanding Dynamic and Static Routing
+
+![Hero image](/engineering-education/understanding-dynamic-and-static-routing/hero.jpeg)
+
+[Source](https://unsplash.com/photos/IL1qSqEMNBo)
+
+## Introduction
+
+Different devices work together to ensure that data is transferred from one end device to another, but the router is the primary device that learns which paths are available and which path is best to forward traffic. The mechanism through which the router makes such a decision is known as routing. The path or route through which a packet is sent can either be a static or dynamic route.
+In this article, we will look at static and dynamic routing in details and it is expected that at the end of this article, the reader should know:
```

Comment:
Grammar mistakes here, at the most basic level, the active voice emphasizes the person or agent who performs an action, in short, the “actor.” The passive voice emphasizes the recipient of the action or sometimes the action itself.
In this, you've used a lot of passive voice which is not recommended in articles.
Kindly make changes where applicable to ensure the article is in the active voice. 
- Feel free to keep in touch with me on Slack inbox.

Reply:
@jumamiller , I could not find you on Slack. please what is your username on Slack?

Reply:
Hi, have you joined the **#eng-ed** group on slack?

Reply:
Yes I have @jumamiller

Reply:
With @Rabo James Bature

Reply:
What's your username? I couldn't find you either

Reply:
Join [here](https://join.slack.com/t/sectionio-community/shared_invite/zt-6174cklt-YFpHmQIAUmc927jqN2EIZQ)

Reply:
@Rabo_James_Bature